### DISCUSSION TITLE
How do I configure a newly brought domain to work with my Vercel application?

### DISCUSSION THREAD

**Discussion by DyloG783** [vercel/community#5512] [Help] [answered]
### Summary

I have found it easy enough to setup a hobby project with Vercel and deploy my application to it. But I decided to buy a domain and I can't understand the documentation to configure it to work with my current application. Can I please get some basic pointers to help me get started?
<img width="232" alt="error1" src="https://github.com/vercel/community/assets/48420066/3272a90c-78f5-4769-80b9-7a20d8957948">
<img width="312" alt="error2" src="https://github.com/vercel/community/assets/48420066/a3d2fe5d-b755-4c07-85a5-eec3ae5471e3">

### Steps to Reproduce

1. have a working hobby production project deployed using vercels free ~domain [something].vercel.app.
2. Buy a domain.
3. Read documentation on how to set your project working with the new domain. Then cry.

**Comment by amyegan** [comment, marked as answer]
Hi @DyloG783! I checked the DNS records for `gamerbuddy.com` that domain, and it looks like it's not configured for a Vercel project.
- [gamerbuddy.com A record](https://www.whatsmydns.net/#A/gamerbuddy.com)
- [gamerbuddy.com CNAME record](https://www.whatsmydns.net/#CNAME/gamerbuddy.com)
- [gamerbuddy.com NS record](https://www.whatsmydns.net/#NS/gamerbuddy.com)

Please follow the steps in our documentation to [add and configure a domain](https://vercel.com/docs/projects/domains/add-a-domain#add-and-configure-domain) and try the tips on [troubleshooting domains](https://vercel.com/docs/projects/domains/troubleshooting#) if you run into more problems.

https://vercel.com/docs/projects/domains/add-a-domain

**Reply by DyloG783** [reply]
Thanks @amyegan this is working now,

What happened is an embarrassing and laughable mistake. In my excitement I purchased an inferior domain name with a dash between the words. So when I went to create these new domains it wouldn't work. I've brought a lesser name and and am stuck with it.

I do feel the documentation is a bit lite. However that is partly due to my lack of experience making it hard to understand documentation in general.

Thanks for the support and fast response!
